v2/factions.json

```diff
@@ -2,13 +2,15 @@
   {
     "code": "adam",
     "color": "000000",
+    "description": "",
     "is_mini": true,
     "name": "Adam",
     "side_code": "runner"
   },
   {
     "code": "anarch",
     "color": "FF4500",
+    "description": "It doesn’t have to be this way.\nAfter centuries of corporate control, the laws of the market seem as real and unbreakable as the laws of physics. When you can’t bear living “sensibly” anymore, there is no choice but to attempt the impossible. And sometimes, it works. It works because we’re desperate, it works because we’ve been lied to. It works because anything is possible.\nThe passions that drive Anarchs are as diverse as the disruptive tactics they employ. To the corp, they are the enemy no plan survives contact with. Anarchs are unpredictable, pushing their limits, they run for a better future.",
```

Comment:
These are fun. Where are they from?

Reply:
https://nisei.net/players/learn-to-play/learn-to-play-runner/ and https://nisei.net/players/learn-to-play/learn-to-play-corp/ ! We don't have any original text for the minis yet, but that's a solvable problem.